src/routes/v0/stream/users.rs

```diff
@@ -92,7 +99,10 @@ pub async fn stream_users_handler(
         Some(skip),
         Some(limit),
         source.clone(),
+        query.source_reach,
         query.depth,
+        Some(timeframe),
+        query.preview,
     )
     .await
```

Comment:
If it does not have any influencers in the selected `timeframe`, it throws a _500_ error which it should be _404_ type

Reply:
It's using the same function we already have? I don't think so as this function is returning a None if the inner user id list is  empty. Am I missing something?

Reply:
ok fixed it.